### PR TITLE
Update vagrant box

### PR DESCRIPTION
**Pull request by bastilian**
Fedora 28 is not available anymore as box this bumps it to the latest version 32.
It also adds `podman` as package to install for convenience.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -20,11 +20,14 @@ Vagrant.configure(2) do |config|
...
-        sudo dnf install -y cockpit
+        sudo dnf install -y cockpit podman
 
         printf "[WebService]\nAllowUnencrypted=true\n" > /etc/cockpit/cockpit.conf
 
         systemctl enable cockpit.socket
         systemctl start cockpit.socket
+
+        systemctl enable podman
+        systemctl start podman
```
Comment (martinpitt) [inline Vagrantfile:31]: Shouldn't that be `io.podman.socket`? The .service unit shouldn't have an `[Install]` section in the first place, that's a bug in podman. Do you know what the difference is between the two sockets? Is podman.socket the new-style REST API (which current master doesn't support yet, @marusak is working on it), and io.podman.socket the varlink API?

Comment (marusak) [reply Vagrantfile:31]: Indeed, `podman.{socket,service}` is the new REST API that is not yet supported and `io.podman.{socket,service}` is the varlink one, that we currently support.

File: Vagrantfile [hunk 2]
```diff
@@ -20,11 +20,14 @@ Vagrant.configure(2) do |config|
...
-        sudo dnf install -y cockpit
+        sudo dnf install -y cockpit podman
 
         printf "[WebService]\nAllowUnencrypted=true\n" > /etc/cockpit/cockpit.conf
 
         systemctl enable cockpit.socket
         systemctl start cockpit.socket
+
+        systemctl enable io.podman.{socket,service}
+        systemctl start io.podman.{socket,service}
```
Comment (martinpitt) [inline Vagrantfile:31]: Please only enable the .socket. The .service really shouldn't have an `[Install]` section in the first place (this is a podman bug).

**Comment by bastilian** [comment]
@martinpitt Thank you for reviewing! It now enables only the socket unit.

**Pull request merged**